psi4/driver/driver.py

```diff
@@ -82,6 +82,9 @@ def _find_derivative_type(ptype, method_name, user_dertype):
             raise ValidationError("_find_derivative_type: user_dertype should only be None or int!")
         dertype = user_dertype
 
+    if (core.get_global_option('INTEGRAL_PACKAGE') == 'ERD') and (dertype != 0):
+        raise ValidationError("INTEGRAL_PACKAGE ERD does not play nicely with gradients, so stopping.")
+
```

Comment:
This is in `_find_dertivative_type` which should be hit by grad() or hess() calls, I think. The language certainly needs generalizing, though, you're right. Will see about LRC check, too. Would be nice to get a clean ERD test suite when `integral_package erd` and `ValidationError` --> `sys.exit(0)`.